commands/call.js

```diff
@@ -32,5 +32,6 @@ async function scheduleFunctionCall(options) {
         options.gas,
         utils.format.parseNearAmount(options.amount));
     const result = providers.getTransactionLastResult(functionCallResponse);
-    inspectResponse.prettyPrintResponse(result, options);
+    inspectResponse.prettyPrintResponse(functionCallResponse, options);
+    console.log(inspectResponse.formatResponse(result));
```

Comment:
shouldn't this happen inside of `inspectResponse.prettyPrintResponse`? Seems like applies to every call that involves sending transaction.

Reply:
Not all the code sending txns was calling getTransactionLastResult and printing it. Do we want to change it?

Reply:
got it, seems like we don't want to change.